py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -786,7 +788,7 @@ def contains_metadata(gm: torch.fx.GraphModule) -> bool:
             require_full_compilation=settings.require_full_compilation,
         )
 
-    dryrun_tracker.unsupported_ops = supported_ops.unsupported_operators
+    # dryrun_tracker.unsupported_ops = supported_ops.unsupported_operators
```

Comment:
Should we should condition this on debug=True?

Reply:
actually this wasn't supposed to land.